pkg/jwt/jwt.go

```diff
@@ -83,11 +85,14 @@ func (t *TokenService) DecodeToken(token string) (*TokenContext, error) {
 		UserID:         claims["UserID"].(string),
 		UserName:       claims["UserName"].(string),
 		UserEmail:      claims["UserEmail"].(string),
-	}, nil
+		UserGroups:     strings.Split(claims["UserGroups"].(string), ","),
```

Comment:
I believe that `UserGroups` will be `[]string{""}` here for non-admin users. Can we clean that up?

Reply:
you're right. cleaned it up by calling slices.DeleteFunc.